doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,189 @@
+"""
+=======================================
+Interactive Tools and Apps with HoloViz
+=======================================
+
+This tutorial shows how to build interactive tools and applications

```

Comment:
```suggestion
This tutorial shows how to build interactive dashboards and applications

```

Reply:
I've changed. But I would actually like to change it back as "tool" is important. To me a tool is some smaller interactive component that you can use in a notebook or in a web application. For your self or your colleagues. And Panel is really good at that. That is one of the reasons is my preferred framework over the competitors. A dashboard is a very visual application with some interactivity. And application can be a dashboard. But it could also be a chat application or something 3rd. Panel is actually really great for smaller chat applications :-).